benchmark/tensorflow-runtime/allreduce_keras_example/exec.py

```diff
@@ -0,0 +1,45 @@
+# allreduce training example
+from __future__ import absolute_import, division, print_function, unicode_literals
+import tensorflow_datasets as tfds
+import tensorflow as tf
+import os
+
+strategy = tf.distribute.experimental.MultiWorkerMirroredStrategy()
+tfds.disable_progress_bar()
+tf.compat.v1.disable_eager_execution()
+
+BUFFER_SIZE = 10000
+BATCH_SIZE = 64
+
+def scale(image, label):
+  image = tf.cast(image, tf.float32)
+  image /= 255
+  return image, label
+
+datasets, info = tfds.load(name='mnist',
+                           with_info=True,
+                           as_supervised=True)
+
+train_datasets_unbatched = datasets['train'].map(scale).cache().shuffle(BUFFER_SIZE).repeat()
+
+def build_and_compile_cnn_model():
+  model = tf.keras.Sequential([
```

Comment:
Since we are using the MNIST dataset, better to use the same Keras model in model_zoo/mnist_functional_api/ for comparison.